openoa/plant.py

```diff
@@ -148,7 +148,7 @@ def _compose_error_message(
 @logged_method_call
 def frequency_validator(
     actual_freq: str | int | float | None,
-    desired_freq: Optional[str | None | set[str]],
+    desired_freq: str | None | set[str] | None,
```

Comment:
None appears twice. Is this a typo?

Reply:
Good catch! I must've missed that one when reviewing.